In [9]:
from nltk.tokenize import sent_tokenize
from random import shuffle

In [2]:
# grab 80 lines (paragraphs) from the file processed by Wiki.Extractor

f = open(r"fiwiki.txt", "r")
lines = []
for line in f.readlines():
    if len(lines) < 80:
        lines.append(line.rstrip())
f.close()
print(lines)

['', 'Ax a di Apasl:', 'Di "h"Ax a di Apasl adawaiz kaal "h"Ax ah wahn a di buk dem a di Baibl ahn ah di falarop to .', '', '1907 Kinston oertkwiek:', 'Di 1907 Kinston oertkwiek we shiekop di kiapital a di ailan a Jumieka wid magnityuud 6.5 pah di muoment magnityuud skiel pah Monde 14 Janiweri, about 3:30 pm luokal taim (20:36 UTC), eh-kansida bai nof raita a di taim fi bi wan a di wol dedlies oertkwiek rikaad ina ischri. Ebri bildin ina Kinston get damij bai di oertkwiek ah sobsikuent fayadem, we laas fi chrii howa bifuo hefoert eh-mek fi put dem out. Ikolminiet ina di det a 800 tu 1,000 piipl, ah lef apraximetli 10,000 uomles ah $25,000,000 ina matiirial damij. Likl afta, wah sunaami eh-ripuot pah di naat kuos a Jumieka, wi maximom wiev ait about 2 m (6–8 ft).', '', '2010 Kinston anres:', 'Di 2010 Kinston anres, dob luokali az di Tivoli Ingkoerjan, wena aam kanflik bitwiin di Showa Pasi jrog kyaatel ah Jumieka milichri ah poliis fuos ina Kinston, di kiapital a Jumieka. Di kanflik eh-

In [3]:
# remove empty lines
for i in lines:
    if "" in lines:
        lines.remove("")

In [4]:
# length without emplty lines
print(len(lines))

56


In [27]:
# 
paras = [[i] for i in lines]
shuffle(paras)
print(paras)

[['Aagianik kimischri a wah kimischri sobdisiplin we invalv di sayantifik stodi a di chokcha, prapati, ah riakshan a aagianik kompoun ah aagianik matiirial, i.e., mata ina vierios faam we kantien kaaban atom.'], ['Aajentiina, ofishali niem di Aajentiin Ripoblik, a di sekan bigis konchri ina Sout Amoerka, an a wah fedarieshan a 23 pravins an wah aatanamos siti, Buenos Airiz. Aajentiina a di iet-bigis konchri ina di wol bai lan ieria ah di bigis outa aal a di konchridem we ab Panish az ofishal languij, alduo Mexiko, Kolombia ah Spien ab muo piipl.'], ['Aajentiina:'], ['Abuja a di kyapital fi Naijiiria.'], ['1907 Kinston oertkwiek:'], ['Abalishanizam wehna muuvment fi "h"en di slieb chried ahn "h"imansipiet di sliebdem ina westan Yuurop ahn di Amorkadem. Nobadi moch no protes di slieb sistim tel di 18t senchri, wen rashanalis tinkadem a di "h"Inlaitnment kritisaizi fi vayaliet yuuman rait, ahn Kwieka ahn ada els ivanjelikal rilijos gruup kandemi az an-Kristian. Duo antisliebri sentiment w

In [39]:
with open('paras.txt', 'w') as f:
    for item in paras:
        for i in item:
            f.write("%s\n" % i)

In [44]:
with open("paras.txt", 'r') as f:
    file_content = open("paras.txt").read()
    sentences = sent_tokenize(file_content)
for sentence in sentences:
    print(sentence)

Aagianik kimischri a wah kimischri sobdisiplin we invalv di sayantifik stodi a di chokcha, prapati, ah riakshan a aagianik kompoun ah aagianik matiirial, i.e., mata ina vierios faam we kantien kaaban atom.
Aajentiina, ofishali niem di Aajentiin Ripoblik, a di sekan bigis konchri ina Sout Amoerka, an a wah fedarieshan a 23 pravins an wah aatanamos siti, Buenos Airiz.
Aajentiina a di iet-bigis konchri ina di wol bai lan ieria ah di bigis outa aal a di konchridem we ab Panish az ofishal languij, alduo Mexiko, Kolombia ah Spien ab muo piipl.
Aajentiina:
Abuja a di kyapital fi Naijiiria.
1907 Kinston oertkwiek:
Abalishanizam wehna muuvment fi "h"en di slieb chried ahn "h"imansipiet di sliebdem ina westan Yuurop ahn di Amorkadem.
Nobadi moch no protes di slieb sistim tel di 18t senchri, wen rashanalis tinkadem a di "h"Inlaitnment kritisaizi fi vayaliet yuuman rait, ahn Kwieka ahn ada els ivanjelikal rilijos gruup kandemi az an-Kristian.
Duo antisliebri sentiment wehn waidpred bai liet 18t se

MorphAdorner's default sentence splitter

1	Aagianik kimischri a wah kimischri sobdisiplin we invalv di sayantifik stodi a di chokcha, prapati, ah riakshan a aagianik kompoun ah aagianik matiirial, i.e., mata ina vierios faam we kantien kaaban atom.
2	Aajentiina, ofishali niem di Aajentiin Ripoblik, a di sekan bigis konchri ina Sout Amoerka, an a wah fedarieshan a 23 pravins an wah aatanamos siti, Buenos Airiz.
3	Aajentiina a di iet-bigis konchri ina di wol bai lan ieria ah di bigis outa aal a di konchridem we ab Panish az ofishal languij, alduo Mexiko, Kolombia ah Spien ab muo piipl.
4	Aajentiina: Abuja a di kyapital fi Naijiiria.
5	1907 Kinston oertkwiek: Abalishanizam wehna muuvment fi "h" en di slieb chried ahn "h" imansipiet di sliebdem ina westan Yuurop ahn di Amorkadem.
6	Nobadi moch no protes di slieb sistim tel di 18t senchri, wen rashanalis tinkadem a di "h" Inlaitnment kritisaizi fi vayaliet yuuman rait, ahn Kwieka ahn ada els ivanjelikal rilijos gruup kandemi az an-Kristian.
7	Duo antisliebri sentiment wehn waidpred bai liet 18t senchri, iab likl imidiet ifek pahn di sentadem a sliebri iself — di Wes Indiz, Sout Amorka, ahn di sodorn Yunaitid Stiet.
8	Di "h" impuotieshan a "h" Afrikan slieb wehn ban ina di British kalanidem ina 1807, ahn ina di Yunaitid Stiet ina 1808.
9	Ina di British Wes Indiz, sliebri abalish ina 1827 ahn ina di French pozeshandem 15 ier lieta.
10	Ina fizix, aabit a di gravitieshanal koerv paat a wah abjek rong wah paint ina spies, fi egzampl di aabit a planit rong wah staar, ar nachral satilait rong wah planit.
11	Aabit a planit tipikali iliptikal, ah di senchral mas we aabit de a di fuokal paint a di ilips.
12	Korant andastandin a di mikianix a aabital muoshan bies pah Alboert Ainstain jinaral tiiri a relitiviti, we akount fi graviti az juu tu koervacha a spies-taim, wid aabit we fala jiodiisix.
13	Fi iiz a kialkiulieshan, relitiviti kamanli apraximiet bai di fuos-bies tiiri a yunivoersal gravitieshan bies pah Kepla laa a planiteri muoshan.
14	

Adolt: Itla wena dekariet vechran a Wol Waar II, ah ih jain di Naatsi Paati ina 1920 ah bikomz di liida ina 1921.
15	Afta ih get lakop fi a fiel kuu ina 1923, ih gien supuot bai promuot Joerman nashinalizim, anti-semitizim, ah anti-kamiunizim wid kiarizmatik arachri ah prapagianda.
16	Ih get apaint chansla ina 1933, ah kwik-taim extablish ah mek rialiti ih vijan a tuotalitierianizim, aatokratik, singgl paati, nashinal suoshalis diktietaship.
17	Itla poersyuu farin palisi wid di diklier guol fi siiz "Lebensraum" ( "" libn spies "") fi Joermani, ah dairek di rizuos a di stiet tuwaad dis guol.
18	Ih riibil di Wehrmacht (defens fuos) ah invied Puolan ina 1939, we liid tu di outbriek a Wol Waar II ina Yuurop.
19	Aatifishal intelijens: Di Aatik a wahn riijan we de roun di Naat Puol a di Ort, opozit di Antaatik riijan roun di Sout Puol.
20	Di aatik inkluud di Aatik Uoshan, an som paat a Kyanada, Griinlan (we Denmaak uon), Rosha, di Yunaitid Stiet (Alaska), Aislan, Naawie, Swiidn, ahn Finlan.
21	Aamiinia: Abaashan a di henin a prignansi bai rimuuv a fiitos ar embrio bifuo ikiah sovaiv outsaida di yuutaros.
22	Abaashan we okor spantienosli nuo az miskiarij.
23	Abaashan kiah kaaz fi poerpos ah den ikaal injuus abaashan, ar les friikuentli, "injuus miskiarij".
24	Di wod "abaashan" noftaim yuuz fi miin onggl injuus abaashan.
25	Simila prosiija afta di fiitos kuda eh potenshal sovaiv outsaida di wuum nuo az "liet toerminieshan a prignansi".
26	Abejar, Soria, Spien: Di huolis faam a haat we dakiument a vijual aat, we ingkluud krieshan a himij ar abjik ina fiil we hingkluud pientin, skolpcha, printmekin, fotografi ah adaels vijual miidia.
27	Aachitecha muotaim ingkluud az wan a di vijual aat; ousomeba, laka dekorietiv aat, ar advataizin, ihinvalv di krieshan a habjik we praktikal kansidarieshan a yuus isenshal — ina wie we no yuujal ina pientin, fi egzampl.
28	Myuuzik, tieta, flim, daans, ah hadaels pofaamin aat, azwel lichicha ah hadaels miidia laka intaraktiv miidia, deh ingkluud ina braada difinishan a haat ar di haats.
29	Optel di 17t senchri, haat kuda eh-refa tu heni skil ar maastari ah neh difranshiet frah kraaf ar sayans.
30	Ina madan yuusij afta di 17t senchri, we hestetik kansidarieshan paramount, fain aat get separiet out ah distingguish frah skil we akuaya jinarali, laka dekorietiv ar aplai haat.
31	Aabit: Pan di neda an, di egzak aada a di kemikal iven tdem we liid tu di fos nyuukliik asid stil a wahn mischri.
32	Nof difrent aipatisis bout orli laif mek op aredi; di muos kaman wan dem a di aiyan-solfa worl tiyori (metabolizm widout jinetiks) ahn di RNA worl aipatisis (RNA laif faam) Ina Britn, Wiliam Wilbafuos (William Wilberforce) wehn tek aan di kaaz a abalishan ina 1787 afta di faamieshan a di Komiti fi di Abalishan a di Slieb Chried, ush ihn liid ina Paalimenchri kiampien fi abalish di slieb chried ina di British Empaya wid di Slieb Chried Ak 1807.
33	Ihn alzwel kiampien fi di abalishan a sliebri ina di British Empaya, ahn ihn lib fi siit ina di Sliebri Abalishan Ak 1833.
34	Di "h" Ax a di Apasl adawaiz kaal "h" Ax ah wahn a di buk dem a di Baibl ahn ah di falarop to.
35	Aakeschra: Di 2011 Jumiekan jinaral ilekshan eh-huol pah 29 Disemba 2011 ina Jumieka.
36	Di ilekshan eh-kantes mienli bitwiin di nieshan tuu mieja palitikal paati, di gobanin Jumieka Lieba Paati (JLP), liid bai Anju Uolnis, ah di apazishan Piipl Nashinal Paati (PNP) liid bai Puosha Simsn-Mila.
37	Di rizolt ena lanslaid vikchri fi di PNP we win 42 a di 63 siit, a tuu-tod majariti.
38	Aakeschra a laaj inschrumental ohsambl yuuz ina klasikal myuuzik we kantien sekshan a chring (vailin, vyuola, chelo ah dobbl bies), braas, udwin, ah pokoshan inschrument.
39	Abuja: Aachitecha (Latn architectura, frah di Griik ἀρχιτέκτων arkhitekton "aachitek", frah ἀρχι - "chiif" ah τέκτων "bilda") a buot di pruoses ah di prodok a planin, dizainin, ah kanschrokin bildin ah ada fizikal chokcha.
40	Ina di nachral saiyans dem, abaiyojenisis, ar di arijin a laif, a di stodi a ou laif pan di ort kuda eva divelop outa non-livin mata.
41	Ano di siem ting az evaluushan, faa dat a di stodi a ou difrent gruup a livin tingz chienj uova taim.
42	Amiino asid, binuon az di "bildin blak fi laif", okor ina niecha, chuu kemikal riakshan we no rilietid to laif.
43	Ina evri livin ting, amiino asid aaganaiz ina pruotiin; di kanschrokshan a dem ya pruotiin kom bout chuu di akshan a nyuukliik asid.
44	Dat go fi miin se di kwestyan bout ou laif pan di ort kom bout afi du wid ou di fos nyuukliik asid kom bout.
45	Iebriam: Di Adisi (Griik: "Ὀδύσσεια" Odýsseia, pronongx [o.dýs.sej.ja] ina Klasikal Atik) a wan a tuu mieja ienshent Griik epik puoym achribiut tu Uoma.
46	Di Adisi a paatli siikuel tu di Iliad, di eda wok we askraib tu Uoma.
47	Di puoym fondamental tu di madoern Westan kianan, an a di sekan uolis extant wok a Westan lichicha, di Iliad a di uolis wan.
48	Skala biliiv se iweh kompuoz nier di "h" en a di 8t senchri BCE, somwe ina Ayuonia, di Griik kuoastal riijan a Anatuolia.
49	Abalishanizam: Adalf Itla, nuo az Adolf Hitler (Joerman pronongxieshan: ˈhɪtlɐ, 20 Iepril 1889 – 30 Iepril 1945), wena Aschrian-baan Joerman palitishan ah di liida a di Nashinal Suoshalis Joerman Woka Paati (Joerman: Nationalsozialistische Deutsche Arbeiterpartei, abriviet NSDAP), papiula nuo az di Naatsi Paati.
50	Ih wena di ruula fi Joermani frah 1933 tu 1945, saab az chansla frah 1933 tu 1945 ah az ed a stiet (Führer und Reichskanzler) frah 1934 tu 1945.
51	Aafwe Chrii, Sint Anju, Jumieka: 2011 Jumiekan jinaral ilekshan: Aat: Di niecha a haat, ah rilietid kansep laka krietiviti ah intopritieshan, expluor ina branch a filasafi nuo az estetix.
52	Aat a wah difrah-difrah rienj a yuuman aktiviti fi kriet vijual, aaditeri ar pofaamans aatifak – aatwok, we hexpress di aata himajinietiv ar teknikal skil, inten fi bi aprishiet fi deh byuuti ar imuoshanal powa.
53	Ina deh muos jinaral faam dem aktiviti ya hingkluud di prodokshan a wok a haat, di kritisizim a haat, di todi a di ischri a haat, ah di hestetik diseminieshan a haat.
54	Iebriam de ina di Buk a Jenisis an a di fos piechriaak a di Izrelait, di Ishmelait, di Midianait ah di Idomait piipldem.
55	Ih rigyaad az di piechriaak a Juu, Kristian ah Muzlim piipl ah di smadi uu ep difain monotiyizim.
56	Iebriam wena di tent jinarieshan frah Nuowa ah di 20t frah Adam.
57	Ih faada wena Tera, an ih bredadem wena Niegor ah Haran.
58	Askaadn tu Jenisis, Gad sen Iebriam fran ih yaad ina Ur Kasdim ah Haran tu Kienan, di lan we Yaawe pramis tu ih disendantdem.
59	Deso, Abraham enta ina wan kovnant: ina exchienj fi rekanaiz Yaawe az Gad, im uda get bles wid nof pitni ah di lan uda bilangx tu dem.
60	Juudizim, Kristianiti, ah Izlam muotaim refa tu az di Iebriamik rilijandem, kaa Iebriam plie mieja ruol ina aal a dem uoli buk.
61	Abaashan: Saiyantis don figa out som tingz bout di arijin a laif, bot som ada wan a stil di sobjek a korent risaach.
62	Di fos livin ting dem pan di ort a wehn prabli singgl sel pruokariyuot.
63	Di oules fasl maikruob-laik objek dem diet bak tu 3.5 Ga (bilian ier uol), jos a fyuu onjrid milian ier yongga dan di ort.
64	Bai 2.4 Ga, di riesho a stiebl aisotup fi kaaban, aiyan, ahn solfa shuo di akshan a livin tingz pahn inaagyanik minaral ahn sediment ahn di molekyula baiyomaaka dem indikiet fuotosintesis, we shuo se laif pahn ort wehn kaman bai da taim ya.
65	Di 2010 Kinston anres, dob luokali az di Tivoli Ingkoerjan, wena aam kanflik bitwiin di Showa Pasi jrog kyaatel ah Jumieka milichri ah poliis fuos ina Kinston, di kiapital a Jumieka.
66	Di kanflik eh-bigin pah di 23 Mie 2010 az sikioriti fuos taat fi saach fi mieja jrog laad Kristifa "Dudus" Kuok, afta di Yunaitid Stiet rikues ih exchradishan, ah di liida a di kriminal giang we atak sebral poliis stieshan.
67	Di voilens, we muosli tek plies uoba 24 – 25 Mie, kil akliis 73 sivilian ah wuun akliis 35 muo.
68	Fuo suoja / poliis get kil tu ah muo dah 500 ares eh-mek, az Jumiekan poliis ah suoja fait gonman ina di Tivoli Gyaadn dischrik a Kinston.
69	Adisi: Aafwe Chrii a di kiapital fi Sint Anju Parish.
70	Aatifishal intelijens (AI) a di intelijens egzibit bai mashiin ar saafwier.
71	A di niem azwel fi di akademik fiil a stodi we stodi ou fi kriet kompyuuta ah kompyuuta saafwier we kiepebl a intelijent biyevia.
72	Mieja AI risaaacha ah texbuk difain dis fiil az "di stodi ah dizain a intelijent iejent", we intelijent iejent a sistim we kiah rejista di invairament ifain iself ina ah tek akshan we maximaiz di chaans a sokses.
73	Jan Makaati (John McCarthy), uu kain di toerm ina 1955, difaini az "di sayans ah injinierin fi mek intelijent mashiin".
74	Adalf Itla: Abraham Lincoln wehna di sixtiint (16t) president a di Yunaitid Stiets.
75	Im liid di konchri outa di bigis intornal kraisis I eva go chuu, di Amerikan Sivl Waar, we prizaab di yuuniyan an en slievri.
76	Bifuo im get ilektid ina 1860 az di fos Ripoblikan president, Lincoln wehna wan laaya, Illinaai stiet lejislieta, wan memba a di Yunaitid Stiet Ous a Reprizentitiv, an wan tuu taim ansoksesful kyandidet tu di U.S. Senit.
77	Im a when wan buol opuonent a di expanshan a slievri ina di Yunaitid Stiets.
78	Lincoln win di Ripoblikan paati naminieshan ina 1860 an get ilek az president lieta da siem ier de.
79	Im taim ina afis akyupai muosli wid di siseshanis Kanfedaret stiet dem fram di sivl waar.
80	Im inchraduus meja fi abolish slievri, chuu di Imansipieshan Praklamieshan ina 1863 an di promuoshan a di Tortiint Amenment tu di kanstityuushan.
81	Di siem taim wen di Sivl Waar out fi don, Lincoln bikom di fos Amerikan president fi get asasiniet.
82	Abraham Lincoln kansistenti rangk bai skala az wan a di grietis U.S. Prezident dem.
83	Wel-nuoon Abalishanisdem inkluudn buot blak piipl, ahn wait piipl we nehn sopuot sliebri: Aat kiah kiaraktaraiz ina toermz a mimiisis (di riprizentieshan a rialiti), expreshan, komiunikieshan a himuoshan, ar adaels kualiti.
84	Juurin di Ruomantik piiriad, aat eh-komiin laka "speshal fakolti a di yuuman main fi klasifai wid rilijan ah sayans".
85	Duo di difinishan a wa ah wa kanstityuut aat dispyuut ah chienj-chienj uoba taim, jinaral diskripshan menshan di aidie a himajinetiv ar teknikal skil we stem frah yuuman iejensi ah krieshan.
86	Aamiinia (Aamiinian: "Հայաստան", chr.
87	"Hayastan", IPA:), ofishali di Ripoblik a Aamiinia (Aamiinian: "Հայաստանի Հանրապետություն", chr.
88	"Hayastani Hanrapetut’yun"), a wah savrin stiet ina di Sout Kaakasos riijan a Yuurieja.
89	Lokiet ina Westan Ieja, ibaada bai Toerki tu di wes, Jaaja tu di naat, di de facto indipendant Nagorno-Karabah Ripoblik ah Azabaijan tu di hiis, ah Iran ah di exkliev a Nakhchivan tu di sout.
90	Aatik: Abraham Lincoln: Ax a di Apasl: Abejar a wahn miunisipaliti luokiet ina di pravins a Soria, Kiastiil ahn León, Spien.
91	Askaadn tu di 2007 sensos (INE), di miunisipaliti ab papyulieshan a 405 inabitant.
92	2010 Kinston anres: Aagianik kimischri: Abaiyojenisis: Aachitecha: Di 1907 Kinston oertkwiek we shiekop di kiapital a di ailan a Jumieka wid magnityuud 6.5 pah di muoment magnityuud skiel pah Monde 14 Janiweri, about 3: 30 pm luokal taim (20: 36 UTC), eh-kansida bai nof raita a di taim fi bi wan a di wol dedlies oertkwiek rikaad ina ischri.
93	Ebri bildin ina Kinston get damij bai di oertkwiek ah sobsikuent fayadem, we laas fi chrii howa bifuo hefoert eh-mek fi put dem out.
94	Ikolminiet ina di det a 800 tu 1,000 piipl, ah lef apraximetli 10,000 uomles ah $ 25,000,000 ina matiirial damij.
95	Likl afta, wah sunaami eh-ripuot pah di naat kuos a Jumieka, wi maximom wiev ait about 2 m (6 – 8 ft).